test/run-test.ps1

```diff
@@ -0,0 +1,65 @@
+[cmdletbinding()]
+param(
+   [string]$OS="windowsservercore"
+)
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference="Stop"
+
+$dockerRepo="microsoft/dotnet-nightly"
+$repoRoot = Split-Path -parent $PSScriptRoot
+
+if ($env:DEBUGTEST -eq $null) {
+    $optionalDockerRunArgs="--rm"
+}
+else {
+    $optionalDockerRunArgs=""
+}
+
+pushd $repoRoot
+
+Get-ChildItem -Directory -Exclude 'test', 'update-dependencies', '.*' | ForEach {
+    $tagBase="$($dockerRepo):$($_.Name)-$OS"
+
+    $timeStamp = Get-Date -Format o | foreach {$_ -replace ":", "." -replace "+", "-"}
```

Comment:
Consider using `Get-Date -Format FileDateTime` instead.  This would be more consistent with the Unix behavior, and would simplify the code, making my previous two comments on this line irrelevant.